internal/server/instance/drivers/driver_qemu.go

```diff
@@ -6470,6 +6487,12 @@ func (d *qemu) updateMemoryLimit(newLimit string) error {
 	if curSizeMB == newSizeMB {
 		return nil
 	} else if baseSizeMB < newSizeMB {
+		limitsMemoryHotplug := d.expandedConfig["limits.memory.hotplug"]
+		memoryHotplugEnabled := !util.IsFalse(limitsMemoryHotplug)
+		if !memoryHotplugEnabled {
```

Comment:
I'd shorten that to just `if util.IsFalse(d.expandedConfig["limits.memory.hotplug"])` so we don't have the double negation and temporary variable.

Reply:
Fixed.